mod-source-record-manager-server/src/main/java/org/folio/services/JobMonitoringServiceImpl.java

```diff
@@ -39,7 +40,7 @@ public Future<List<JobMonitoring>> getAll(String tenantId) {
 
   @Override
   public Future<List<JobMonitoring>> getInactiveJobMonitors(Long maxInactiveInterval, String tenantId) {
-    var lastInactiveDateTime = LocalDateTime.now().minus(maxInactiveInterval, ChronoUnit.MILLIS);
+    var lastInactiveDateTime = LocalDateTime.now(ZoneOffset.UTC).minus(maxInactiveInterval, ChronoUnit.MILLIS);
```

Comment:
I had to make this change due to failing tests in my local env, the PR branches but not in master for some weird reason.